core-server/Core/RestTesterFunctions.py

```diff
@@ -6997,17 +7004,32 @@ def post(self):
         if success != Context.instance().CODE_OK:
             raise HTTP_500("Unable to open test file")
 
-        return { "cmd": self.request.path, 
-                 "file-content": data_base64,
-                 "file-path": path_file,
-                 "file-name": name_file,
-                 "file-extension": ext_file,
-                 "locked": locked,
-                 "locked-by": locked_by,
-                 "project-id": project,
-                 "custom-param": _customParam,
-                 "action-id": _actId,
-                 "destination-id": _destId}
+        # dbr13 >>>
+        # When we set checkbox in the Update->Location
+        if update_location:
```

Comment:
Can you explain why you update linked script path when you just update the location of a test in testplan ?
This change is not clear for me

Reply:
it's more clear after watching again the video.

perhaps, it will be better to have a dedicated ressources in the rest api to  fix links.
just with the code, it is not sufficient to understand properly

Reply:
Maybe you're right. But at that moment I decided in that way =)